terraform/environments/nomis/application_variables.tf

```diff
@@ -89,12 +90,27 @@ locals {
           instance_type          = "r6i.2xlarge"
           asm_data_capacity      = 4000
           asm_flash_capacity     = 1000
-          description            = "Copy of Production NOMIS Audit database in Azure PDPDL00038, replacting with PDPDL00038, a replacement for PDPDL00037."
+          description            = "Copy of Production NOMIS Audit database in Azure PDPDL00038, replicating with PDPDL00038, a replacement for PDPDL00037."
           termination_protection = true
+          oracle_sids            = ["CNMAUDPD"]
           oracle_app_disk_size = {
             "/dev/sdb" = 100  # /u01
             "/dev/sdc" = 5120 # /u02
           }
+        },
+        AUDITa = {
```

Comment:
better to have AUDIT_REPLICA if _ are valid but i'm assured this will be removed

Reply:
Updated the name.